src/main/scala/graphql/HanamuraService.scala

```diff
@@ -0,0 +1,37 @@
+package graphql
+import models.User
+import org.mongodb.scala.MongoCollection
+import zio.{ Fiber, Queue, Ref, Task, UIO, ZIO }
+
+import scala.concurrent.{ ExecutionContext, Future }
+// pass ref of database for future implementation
+class HanamuraService(userCollection: Ref[MongoCollection[User]],
+                      subscribers: Ref[List[Queue[String]]]) {
+  val ec: ExecutionContext = ExecutionContext.global
+  def sayHello: Task[String] =
+    ZIO.fromFuture(
+      implicit ec => Future.failed(new Exception)
+    )
+  def getUserFromDatabase: Task[Seq[User]] =
```

Comment:
Note that caliban currently has Schema for List and Set (not Seq) so you might prefer using a List here, or provide the Schema for Seq (should be the same as List anyway).